bouncer/shared/boost.ts

```diff
@@ -81,13 +82,19 @@ export async function addBoostFunds(
   boostTier: number,
   amount: number,
   lpUri = '//LP_BOOST',
+  depositLiquidity = true,
 ): Promise<Event> {
   await using chainflip = await getChainflipApi();
   const lp = keyring.createFromUri(lpUri);
+  console.log(`address: ${lp.address}`);
 
-  assert(boostTier > 0, 'Boost fee must be greater than 0');
+  assert(boostTier > 0, 'Boost tier must be greater than 0');
 
-  await provideLiquidity(asset, amount, true, lpUri);
+  if (depositLiquidity) {
+    // Provide a little more than the amount because some will be lost due to ingress fee
```

Comment:
I think this could be a little surprising to the caller of `addBoostFunds`. Can we not change the caller to instead provide a bit more than necessary?

Also, do we really need this function to fund the account internally? Shouldn't be too difficult to call `provideLiquidity` first if necessary. Then `depositLiquidity` parameter won't be necessary.

Reply:
> do we really need this function to fund the account internally?

I was trying to make things simple for the tests. But yeah, turned out messy like this. I'll try removing it.

>Can we not change the caller to instead provide a bit more than necessary

I Wanted the added boost funds to still be exact. Thought this was the best way to do it.

Reply:
Relying on the boost funds being exact is probably going to end up in a flaky test, bouncer should be considered to test the overall end-to-end with the exact values of things tested in unit tests in the individual components